Description: Point #include statements to the new location of the V4L1 headers
 as they are no longer provided by the kernel.
Author: Alessio Treglia <alessio@debian.org>
Bug-Debian: http://bugs.debian.org/cgi-bin/bugreport.cgi?bug=621954
Forwarded: https://github.com/unicap/unicap/pull/6

--- unicap.orig/cpi/v4l/v4l.c
+++ unicap/cpi/v4l/v4l.c
@@ -49,7 +49,7 @@
 #endif
 #include <debug.h>
 
-#include <linux/videodev.h>
+#include <libv4l1-videodev.h>
 #include <linux/videodev2.h> // for v4l2 checks
 
 #include "v4l.h"
